src/components/manifold-invoices/manifold-invoices.tsx

```diff
@@ -25,29 +25,26 @@ export class ManifoldInvoices {
   connection: Connection;
 
   async componentWillLoad() {
-    await customElements.whenDefined("manifold-init");
-    const core = document.querySelector(
-      "manifold-init"
-    ) as HTMLManifoldInitElement;
+    await customElements.whenDefined('manifold-init');
+    const core = document.querySelector('manifold-init') as HTMLManifoldInitElement;
     this.connection = await core.initialize({
       element: this.el,
-      componentVersion: "<@NPM_PACKAGE_VERSION@>",
+      componentVersion: '<@NPM_PACKAGE_VERSION@>',
       version: 0,
     });
 
     this.fetchInvoices();
   }
 
   async fetchInvoices() {
-    const res = await this.connection.graphqlFetch<InvoicesQuery>({
-      query,
-      variables: {
-        first: 100,
-        after: "",
-      },
-    });
+    const res = await this.connection.graphqlFetch<InvoicesQuery>(
+      { query, variables: { first: 100, after: '' } },
+      { headers: { 'X-Manifold-Sample': 'Platform' } }
+    );
 
-    this.data = res.data;
+    if (res.data) {
+      this.data = res.data;
+    }
```

Comment:
This was actually added because `strictNullChecks` caught a null reference error